Spokestack/TextToSpeech.swift

```diff
@@ -172,3 +184,15 @@ import AVFoundation
         }
     }
 }
+
+fileprivate struct SynthesizeResponseURL: Codable {
+    let url: URL
+}
+
+fileprivate struct SynthesizeResponseSynthesize: Codable {
```

Comment:
This name's pretty confusing.

Reply:
I don't pick the response structure, and the struct naming scheme seems reasonable 🤷‍♂. Open to suggestions though!

Reply:
What about `TTSResponse` `Body`/`Method`/`URL`?